packages/api/src/command/medical/patient/converage-assessment-create.ts

```diff
@@ -0,0 +1,115 @@
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { chunk } from "lodash";
+import { Patient } from "@metriport/core/domain/patient";
+import { sleep } from "@metriport/shared";
+import { DocumentQueryProgress } from "@metriport/core/domain/document-query";
+import { createPatient, PatientCreateCmd } from "./create-patient";
+import { matchPatient } from "./get-patient";
+import { updatePatient } from "./update-patient";
+import { getPatientOrFail } from "./get-patient";
+import { queryDocumentsAcrossHIEs } from "../document/document-query";
+import { getConsolidated, ConsolidatedData } from "./consolidated-get";
+
+dayjs.extend(duration);
+
+const delayTime = dayjs.duration(30, "seconds").asMilliseconds();
+const dqDrBuffer = dayjs.duration(10, "minutes").asMilliseconds();
```

Comment:
nit: I've been recommending the team to keep those as "duration" and convert to millis on the place that needs those as numbers. Reason being that if we want to use `delayTime` for anything else, its in "duration", not a number that we don't know what's the unit for.

Reply:
Done.